specification/cognitiveservices/data-plane/MetricsAdvisor/preview/v1.0/MetricsAdvisor.json

```diff
@@ -6060,21 +6332,18 @@
       "x-ms-discriminator-value": "Email"
     },
     "WebhookHookParameter": {
-      "required": [
-        "endpoint"
-      ],
       "type": "object",
       "properties": {
         "endpoint": {
           "description": "API address, will be called when alert is triggered, only support POST method via SSL",
           "type": "string"
         },
         "username": {
-          "description": "basic authentication",
+          "description": "(Deprecated) The username, if using basic authentication",
           "type": "string"
         },
         "password": {
-          "description": "basic authentication",
+          "description": "(Deprecated) The password, if using basic authentication",
           "type": "string"
         },
```

Comment:
Why are we deprecating these properties? Should we just completely remove them instead?

Reply:
They are considered as less secure and will be replaced by certificates. While some public preview customers have already using it, so it might not be removed completely.